src/protocol/codec/transport.c

```diff
@@ -28,6 +28,10 @@
 #include "zenoh-pico/utils/logging.h"
 #include "zenoh-pico/utils/result.h"
 
+#define _Z_FRAME_VEC_BASE_SIZE 8  // Abritrary small value
+#define _Z_FRAME_VEC_SIZE_FROM_ZBUF_LEN(len) \
+    _Z_FRAME_VEC_BASE_SIZE + (len) / Z_CONFIG_FRAME_AVG_MSG_SIZE  // Approximate number of messages in frame
```

Comment:
```suggestion
    (_Z_FRAME_VEC_BASE_SIZE + (len) / Z_CONFIG_FRAME_AVG_MSG_SIZE)  // Approximate number of messages in frame
```
To avoid potential incorrect usage in expressions